crates/uv-build/pyproject.toml

```
@@ -0,0 +1,54 @@
+[project]
+name = "uv-build"
+version = "0.5.30-rc.1"
+description = "Build backend only distribution for uv."
```

Maybe?

```suggestion
description = "The uv build backend"
```

It's awkward as phrased currently. I'm not sure we need to explain it's a partial distribution here.